packages/gafl-webapp-service/src/pages/contact/contact/contact.njk

```diff
@@ -120,32 +160,7 @@
           classes: "govuk-body-m"
         }
       },
-      items: [
-        {
-          value: "email",
-          text: "Email",
-          checked: payload['how-contacted'] === 'email',
-          conditional: {
-            html: emailHtml
-          }
-        },
-        {
-          value: "text",
-          text: "Text message",
-          checked: payload['how-contacted'] === 'text',
-          conditional: {
-            html: textMessageHtml
-          }
-        },
-        {
-          value: "none",
-          text: 'Post' if data.isPhysical else 'Make a note of the licence number',
-          checked: payload['how-contacted'] === 'none',
-          conditional: {
-            html: postMessageHtml if data.isPhysical else noneMessageHtml
-          }
-        }
-      ]
+      items: itemsArray
     }) }}
```

Comment:
Ah, it was already in the template, you just moved it. Still not convinced about it being here though!